lib/rules/jsx-uses-react.js

```diff
@@ -14,6 +14,7 @@ const docsUrl = require('../util/docsUrl');
 
 module.exports = {
   meta: {
+    type: 'suggestion',
```

Comment:
i'm not sure if this one should have a fix type; it needs to be run *always* to be sure that variables are properly marked as used.

Reply:
That's the meta type. I think all the rules should have it.

Reply:
I'm saying that this rule shouldn't, because it shouldn't be filtered out of any category. In other words, if i say "only run problems", and this rule doesn't run, things won't work properly.

Reply:
If we do not specify a type, then --fix with --fix-type will ignore this rule. I am not sure that we can specify array of types.
What are your suggestions?

Reply:
If we do not specify a type, then --fix with --fix-type will ignore this rule. I am not sure that we can specify array of types.
What are your suggestions?

Reply:
hmm - maybe we could file a bug upstream with eslint to allow supporting an array of types, or, to include rules with unspecified types?

Reply:
The first idea sounds good, but it's a long time. And this bug can be rejected.
And I'm not sure if this is really necessary.

Reply:
Maybe we specify the type as "error" and that's it?